cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
   }
 
-  typeName(name) {
+  typeName() {
     this.nameField.type(name, { force: true });
```

Comment:
The `typeName` method is missing the `name` parameter. You need to pass `name` as an argument to this method to avoid reference errors.